bindings/rust/publish/publish_fips.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Nit-pick: We use dash (`-`) for `generate-fips.sh`, but underscore (`_`) for `publish_fips.sh`.

Reply:
Changed :)